src/view/configuration/configuration.jsx

```diff
@@ -230,457 +232,456 @@ const validationSchema = object()
     );
   });
 
-const Configuration = () => {
-  const [selectedAccordionIndex, setSelectedAccordionIndex] = useState();
-  const [isFirstExtensionViewRender, setIsFirstExtensionViewRender] = useState(
```

Comment:
The problem was that we were using these react hooks from the `render` function that's fed into `ExtensionView`, which React didn't like (but didn't actually break anything). I split the render code into its own component and moved to hooks into that component. I also leveraged the `useNewlyValidatedFormSubmission` hook I created for the XDM Object data element view, which simplified some of the code as well.